### PR TITLE
a tidy up of the docstrings in quant_api

### PR DESCRIPTION
**Pull request by Sparrow0hawk**
I've gone through [quant_api.py](https://github.com/Urban-Analytics/RAMP-UA/blob/master/microsim/quant_api.py) file to tidy up docstrings into the sphinx format and add more spacing for readability.

### CODE REVIEW

File: microsim/quant_api.py [hunk 1]
```diff
@@ -54,107 +59,173 @@ def read_data(cls,QUANT_DIR):
...
         a list of all the surrounding primary schools whose probabilty of being visited by the MSOA_IZ is
         greater than or equal to the threshold.
         School ids are taken from the Edubase list of URN
         NOTE: code identical to the secondary school version, only with switched lookup tables
-        @param msoa_iz An MSOA code (England/Wales e.g. E02000001) or an IZ2001 code (Scotland e.g. S02000001)
-        @param threshold Probability threshold e.g. 0.5 means return all possible schools with probability>=0.5
-        @returns a list of probabilities in the same order as the venues
-        """
+
+        :param dfPrimaryPopulation: # needs clarification
```
Comment (Sparrow0hawk) [inline microsim/quant_api.py:68]: @nickmalleson be grateful if you could review some of these docstrings as there are parameters I'm unable to properly document (I don't know what they are!)

File: microsim/quant_api.py [hunk 2]
```diff
@@ -54,107 +59,173 @@ def read_data(cls,QUANT_DIR):
...
+        :type dfPrimaryPopulation: pandas.DataFrame
+        :param dfPrimaryZones: # needs clarification
```
Comment (Sparrow0hawk) [inline microsim/quant_api.py:70]: Same as above here

File: microsim/quant_api.py [hunk 3]
```diff
@@ -54,107 +59,173 @@ def read_data(cls,QUANT_DIR):
...
         Given an MSOA area code (England and Wales) or an Intermediate Zone (IZ) 2001 code (Scotland), return
         a list of all the surrounding secondary schools whose probabilty of being visited by the MSOA_IZ is
         greater than or equal to the threshold.
         School ids are taken from the Edubase list of URN
         NOTE: code identical to the primary school version, only with switched lookup tables
-        @param msoa_iz An MSOA code (England/Wales e.g. E02000001) or an IZ2001 code (Scotland e.g. S02000001)
-        @param threshold Probability threshold e.g. 0.5 means return all possible schools with probability>=0.5
-        @returns a list of probabilities in the same order as the venues
+
+        :param dfSecondaryPopulation: # needs clarification
```
Comment (Sparrow0hawk) [inline microsim/quant_api.py:112]: Same as above here

File: microsim/quant_api.py [hunk 4]
```diff
@@ -54,107 +59,173 @@ def read_data(cls,QUANT_DIR):
...
+        :type dfSecondaryPopulation: pandas.DataFrame
+        :param dfSecondaryZones: # needs clarification
```
Comment (Sparrow0hawk) [inline microsim/quant_api.py:114]: Same as above here

File: microsim/quant_api.py [hunk 5]
```diff
@@ -54,107 +59,173 @@ def read_data(cls,QUANT_DIR):
...
-        getProbableRetailByMSOAIZ
         Given an MSOA area code (England and Wales) or an Intermediate Zone (IZ) 2001 code (Scotland), return
         a list of all the surrounding retail points whose probabilty of being visited by the MSOA_IZ is
         greater than or equal to the threshold.
         Retail ids are from ????
-        @param msoa_iz An MSOA code (England/Wales e.g. E02000001) or an IZ2001 code (Scotland e.g. S02000001)
-        @param threshold Probability threshold e.g. 0.5 means return all possible retail points with probability>=0.5
-        @returns a list of probabilities in the same order as the venues
+
+        :param dfRetailPointsPopulation: # needs clarification
```
Comment (Sparrow0hawk) [inline microsim/quant_api.py:153]: Same as above here

File: microsim/quant_api.py [hunk 6]
```diff
@@ -54,107 +59,173 @@ def read_data(cls,QUANT_DIR):
...
+        :type dfRetailPointsPopulation: pandas.DataFrame
+        :param dfRetailPointsZones: # needs clarification
```
Comment (Sparrow0hawk) [inline microsim/quant_api.py:155]: Same as above here

File: microsim/quant_api.py [hunk 7]
```diff
@@ -54,107 +59,173 @@ def read_data(cls,QUANT_DIR):
...
         Given an MSOA area code (England and Wales) or an Intermediate Zone (IZ) 2001 code (Scotland), return
         a list of all the surrounding hospitals whose probabilty of being visited by the MSOA_IZ is
         greater than or equal to the threshold.
         Hospital ids are taken from the NHS England export of "location" - see hospitalZones for ids and names (and east/north)
         NOTE: code identical to the primary school version, only with switched lookup tables
-        @param msoa_iz An MSOA code (England/Wales e.g. E02000001) or an IZ2001 code (Scotland e.g. S02000001)
-        @param threshold Probability threshold e.g. 0.5 means return all possible hospital points with probability>=0.5
-        @returns a list of [ {id: 'hospitalid1', p: 0.5}, {id: 'hospitalid2', p:0.6}, ... etc] (NOTE: not sorted in any particular order)
+
+        :param dfHospitalPopulation: # needs clarification
```
Comment (Sparrow0hawk) [inline microsim/quant_api.py:197]: Same as above here

File: microsim/quant_api.py [hunk 8]
```diff
@@ -54,107 +59,173 @@ def read_data(cls,QUANT_DIR):
...
+        :type dfHospitalPopulation: pandas.DataFrame
+        :param dfHospitalZones: # needs clarification
```
Comment (Sparrow0hawk) [inline microsim/quant_api.py:199]: Same as above here

**Review by Sparrow0hawk — COMMENTED** [review]
Highlighting sections where I need further guidance on what function inputs are

**Review by nickmalleson — APPROVED** [review]

**Comment by nickmalleson** [comment]
All done @Sparrow0hawk following the emails with Richard. Thanks for this!

**Review by Sparrow0hawk — COMMENTED** [review]
Thanks for this @nickmalleson

**Pull request merged**